examples/trace/gcp_exporter/foo_library/foo_library.cc

```diff
@@ -0,0 +1,33 @@
+#include "opentelemetry/trace/provider.h"
```

Comment:
Nit: It might worth having some comments (or readme file) in the file. Like having a graph showing how the spans look like in a trace: 
 -> Span 2========
[space] -> Span 1===
 [space]-> Span 1===

Reply:
Thank you for the suggestion! 

Yes, adding a README file is on my agenda.
I will also add one for the GCP Exporter.